modules/packages/Curl.chpl

```diff
@@ -286,7 +288,8 @@ module Curl {
      :arg args: any number of tuples of the form (curl_option, value).
```

Comment:
Not your fault, but it looks like the documentation for this method doesn't talk about the `ch` arg.  Might be good to add a note about it (though I'd understand if that felt like scope creep, so it'd be okay to not)